__tests__/port.test.js

```diff
@@ -0,0 +1,37 @@
+const Port = require("../src/port");
+
+describe("port", () => {
+  describe("port is instantiated", () => {
```

Comment:
Unnecessary nesting of describe blocks. You only need one of these. If you wanted both I would at least move `let port` and the `beforeEach` outside of the second describe block so that you could make use of that port in other tests